### PR TITLE
add compatibility check badges to README

### PR DESCRIPTION
**Pull request by ylil93**
Hello google package maintainer,
The badges being added to the README in this PR will indicate your compatibility with other google packages. This addresses a set of user bugs which have happened when a user depends on two Cloud libraries (or runtimes that bundle libraries), say A and B, which both depend on library C. If the two libraries require different versions of C, the users can run into issues both when they pip install the libraries, and when they deploy their code. Our compatibility server checks that all libraries we make including this one are self and pairwise compatible as well as not having any deprecated dependencies. The two badges will mark the build for your project green when the latest version available on PyPI and github HEAD respectively meet all compatibility checks with itself and all other libraries. The badge target will link to a details page that elaborates on the current status. This should help you fix issues pre-release, to avoid user surprises. For more information, please take a look at our project charter at go/python-cloud-dependencies-project-charter and the badging PRD https://docs.google.com/document/d/1GYRFrfUou2ssY71AtnLkc8Sg1SD4dxqN4GzlatGHHyI/edit?ts=5c6f031d

R: @robertwb

### CODE REVIEW

**Comment by iemejia** [comment]
Seems the document mentioned in this PR is not publicly accesible.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,8 @@
 [![PyPI version](https://badge.fury.io/py/apache-beam.svg)](https://badge.fury.io/py/apache-beam)
 [![Build Status](https://builds.apache.org/buildStatus/icon?job=beam_PostCommit_Java)](https://builds.apache.org/job/beam_PostCommit_Java)
 [![Coverage Status](https://coveralls.io/repos/github/apache/beam/badge.svg?branch=master)](https://coveralls.io/github/apache/beam?branch=master)
+[![Compat Check PyPI](https://python-compatibility-tools.appspot.com/one_badge_image?package=apache-beam%5Bgcp%5D)](https://python-compatibility-tools.appspot.com/one_badge_target?package=apache-beam%5Bgcp%5D)
+[![Compat Check Github](https://python-compatibility-tools.appspot.com/one_badge_image?package=git%2Bgit%3A//github.com/apache/beam.git%23subdirectory%3Dsdks/python)](https://python-compatibility-tools.appspot.com/one_badge_target?package=git%2Bgit%3A//github.com/apache/beam.git%23subdirectory%3Dsdks/python)
```
Comment (aaltay) [inline README.md]: How about `Compat Check at master` ?

Comment (ylil93) [reply README.md]: done

**Comment by ylil93** [comment]
> Seems the document mentioned in this PR is not publicly accesible.

Publicly accessible equivalents for the project charter and badge PRD, respectively, are the following:
https://github.com/GoogleCloudPlatform/cloud-opensource-python/blob/master/README.rst
https://github.com/GoogleCloudPlatform/cloud-opensource-python/blob/master/badge_server/README.rst

**Comment by ylil93** [comment]
@iemejia 
@aaltay 
Gentle ping on this PR?

**Comment by aaltay** [comment]
LGTM - Merging based on this discussion (https://lists.apache.org/thread.html/d3687c699fd8e05fbee0171ac4ba75924eba72e572aed12dbc597018@%3Cdev.beam.apache.org%3E)

@ylil93 if we encounter issues, how should we report it?

**Pull request merged**